src/components/earn/PoolCardTri/index.tsx

```diff
@@ -70,16 +79,22 @@ const DefaultPoolCardtri = ({
   const history = useHistory()
   const { t } = useTranslation()
 
+  const [showMore, setShowMore] = useState(isMobileOnly && isStaking ? true : false)
```

Comment:
```suggestion
  const [showMore, setShowMore] = useState(isMobileOnly && isStaking)
```

Reply:
If mobile UI is constrained vertically, doesn't it make sense to have everything collapsed?  This is assuming it's easy to tell which pools the user is staked in while the card is collapsed